decidim-core/spec/lib/events/simple_event_spec.rb

```diff
@@ -19,8 +19,7 @@ module Decidim
       let(:resource) { create(:dummy_resource, title: { en: "<script>alert('Hey');</script>" }) }
 
       it "sanitizes the HTML tags from the i18n options" do
-        expect(subject.i18n_options[:resource_title])
-          .to eq "alert('Hey');"
+        expect(subject.i18n_options[:resource_title]).to eq("alert('Hey');")
```

Comment:
This wasn't done in develop. This kind of changes make more difficult to make the backports/reviews/etc 
```suggestion
        expect(subject.i18n_options[:resource_title])
          .to eq("alert('Hey');")
```